Root/Krzysztof/Root/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link href="https://fonts.googleapis.com/css?family=Anton" rel="stylesheet">
+		<link type="text/css" rel="stylesheet" href="lightbox/dist/css/lightbox.css"/>
```

Comment:
Watch indentation, again just for readability. Keeping things consistant when you have larger files is quite important.